docs/docs/tuning-your-model.mdx

```diff
@@ -130,7 +130,11 @@ The image below shows the call order during the training of this pipeline:
 
 
 
-<img alt="Component Lifecycle" src={useBaseUrl("/img/component_lifecycle.png")} />
+<img caption="Component Lifecycle" src={useBaseUrl("/img/component_lifecycle.png")
+alt="The component lifecycle during training. Components are processed in the order
+they're listed in the configuration file.
+All components are created and initialized in order before they are trained in
+order and then persisted in order."} />
```

Comment:
I think you might want to reverse `caption` and `alt` here—and use an `Image` component for this since `caption` won't do anything to an HTML `img` tag. Also same comment as above re versioning